design/ceph/osd-migration.md

```diff
@@ -36,139 +24,102 @@ considered in the future:
   and application data on slower media.
 - Setups with multiple OSDs per drive, though with recent Ceph releases the
   motivation for deploying this way is mostly obviated.
+- Mix of encrypted and unencrypted OSDs in the cluster.
 
 ## Proposal
+- Since migration requires destroying of the OSD and cleaning data from the disk,
+  a user confirmation in the CR is required before proceeding.
+- Operator would identify the OSDs that need migration by looking at the labels on
```

Comment:
How about in a more detailed section below, discuss how OSD deployment labels are used? This statement can stay general. Then you can mention which labels will help identify which scenario is requiring the migration (encryption, store type, etc)
```suggestion
- Operator would identify the OSDs that need migration according to the changed settings
```